### PR TITLE
(PDB-2661) Document the puppetdb_query function

### PR DESCRIPTION
**Pull request by wkalt**
Add a blurb to docs describing usage of puppetdb_query.

### CODE REVIEW

File: documentation/api/query/tutorial-pql.markdown [hunk 1]
```diff
@@ -90,6 +90,17 @@ only accept unencrypted traffic from `localhost`.
 This requires that you specify a certificate (issued by the same CA PuppetDB
 trusts), a private key, and a CA certificate.
 
+### Querying with puppet code
+
+The PuppetDB terminus includes the `puppetdb_query` function, which can be used
+to query PuppetDB from within a Puppet manifest. For example,
+
+    $last_node_query = 'nodes{order by report_timestamp desc limit 1}'
```
Comment (ajroetker) [inline documentation/api/query/tutorial-pql.markdown]: might be helpful to note you can use the AST syntax too, `$last_node_query = [ "from", "nodes", <stuff> ]`

Comment (ajroetker) [inline documentation/api/query/tutorial-pql.markdown]: Also showing an example of how an exported resource query translates to a `puppetdb_query` could be useful

Comment (ajroetker) [inline documentation/api/query/tutorial-pql.markdown]: The last thing that might be interesting for users is showing how you can use this for getting a list of nodes with some property (which you can't do with exported resources).

Comment (wkalt) [inline documentation/api/query/tutorial-pql.markdown]: thanks, these are good suggestions

**Pull request merged**